src/index.html

```diff
@@ -6,17 +6,345 @@
+            </ul>
+
+            <div class="scheduling-info">
+              <p class="number">+1 234 5555-55-55</p>
+
+              <a
+                href="tel:+12345555555"
+                class="schedule-description"
+              >
+                Book a test ride
+              </a>
+            </div>
+          </nav>
+        </div>
+      </div>
+
+      <span>Take the Streets</span>
+    </header>
+
+    <div class="container">
+      <main class="main">
+        <div
+          class="brief-description"
+          id="about"
+        >
+          <h1>Move Free</h1>
+
+          <p class="description">
+            Shift your ride, not gears. Find the fastest way to move in the city
+            as the bike adapts intuitively to power the speed you need.
+          </p>
+        </div>
+
+        <h2
+          class="title"
+          id="compare"
+        >
+          Compare Bikes
+        </h2>
+
+        <section class="bike-card">
+          <article>
+            <a href="$">
+              <img
+                src="./assets/bk-1.png"
+                alt="imagem de uma bicicleta verde"
+                width="100%"
+                loading="lazy"
+              />
+            </a>
+
+            <h3>Sporty 4</h3>
+
+            <p class="bike-description">
+              The iconic frame brought to a new performance height as a sporty,
+              active ride.
+            </p>
+            <p class="bike-price">$ 2590</p>
+          </article>
+
+          <article>
+            <a href="#">
+              <img
+                src="./assets/bk-2.png"
+                alt="imagem de uma bicicleta cinza claro"
+                width="100%"
+                loading="lazy"
+              />
+            </a>
+
+            <h3>Ride in town ST</h3>
+
+            <p class="bike-description">
+              An open frame for an upright riding position as the most
+              comfortable ride in town.
+            </p>
+            <p class="bike-price">$ 2590</p>
+          </article>
+
+          <article>
+            <a href="#">
+              <img
+                src="./assets/bk-3.png"
+                alt="imagem de uma bicicleta branca"
+                width="100%"
+                loading="lazy"
+              />
+            </a>
+
+            <h3>Agile ride 3</h3>
+
+            <p class="bike-description">
+              The lightweight frame that has earned its street tread as a sleek,
+              agile ride.
+            </p>
+            <p class="bike-price">$ 2090</p>
+          </article>
+        </section>
+
+        <section class="details-card">
+          <h2
+            class="title"
+            id="details"
+          >
+            The Details
+          </h2>
+
+          <article>
+            <div class="bike-details">
+              <img
+                src="./assets/details-1.png"
+                alt="Suporte para celular"
+                class="details-image-md-1"
```

Comment:
Modifiers missing the BEM double hyphen:

Modifiers in BEM should use the --modifier syntax.

Example:
details-image-md-1 could be better named as details__image--md-1.